src/main/java/ml/shifu/shifu/core/processor/TrainModelProcessor.java

```diff
@@ -1515,10 +1518,12 @@ public void setForVarSelect(boolean isForVarSelect) {
     private static class TailThread extends Thread {
         private long offset[];
         private String[] progressLogs;
-
-        public TailThread(String[] progressLogs) {
+        private BufferedWriter bw ;
+        private String filePath = "trainErrorLog.csv";
```

Comment:
seems file name is a fix name, do you think it is better to rename to a name with basic#name related file.